tools/test/travis-ci/doxy-spellchecker/ignore.en.pws

```diff
@@ -117,4 +117,6 @@ Hinnant
 Vin
 Vref
 ssid
+instantiation
```

Comment:
Does it cover plurals?

Reply:
No, We should add plurals of the same word to consider by Doxygen spell check

Reply:
Okay :(